app/components/channel_drawer/channels_list/switch_teams_button/switch_teams_button.js

```diff
@@ -14,96 +14,50 @@ import Badge from 'app/components/badge';
 import {wrapWithPreventDoubleTap} from 'app/utils/tap';
 import {changeOpacity, makeStyleSheetFromTheme} from 'app/utils/theme';
 
-export default class SwitchTeams extends React.PureComponent {
+export default class SwitchTeamsButton extends React.PureComponent {
```

Comment:
Cool, I wanted to rename this myself so that I stop confusing it with the SelectTeams screen